frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -143,7 +167,7 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
               </div>
               <div className="co-m-table-grid__body">
                 {_.map(sortedItems, (item) => (
-                  <ItemRow item={item} key={item.uid} />
+                  <ItemRow item={item} key={_.get(item, 'uid')} />
```

Comment:
If we need to use `_.get`, it means not everything has a UID, which means not all rows have keys. That's bad.

Reply:
fixed